sui_types/src/serialize.rs

```diff
@@ -12,6 +13,25 @@ use serde::{Deserialize, Serialize};
 #[path = "unit_tests/serialize_tests.rs"]
 mod serialize_tests;
 
+// SerializedEnvelope wraps application level payloads
```

Comment:
You can use `///` for RustDoc that will show up in the published docs, `//` does not show up in RustDoc.